docs/contributing/docs-contributions.md

```diff
@@ -173,7 +173,8 @@ To write a new GraphQL example, a Codesandbox project with a Gatsby site can be
 ```mdx
 Here's an example of a GraphQL query inline:
 
-<iframe src="https://711808k40x.sse.codesandbox.io/___graphql?query=query%20TitleQuery%20%7B%0A%20%20site%20%7B%0A%20%20%20%20siteMetadata%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false&operationName=TitleQuery" /> // highlight-line
+<iframe src="https://711808k40x.sse.codesandbox.io/___graphql?query=query%20TitleQuery%20%7B%0A%20%20site%20%7B%0A%20%20%20%20siteMetadata%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false&operationName=TitleQuery" /> //
+highlight-line
```

Comment:
Moving `highlight-line` to the new line doesn't look right.

Reply:
this got merged with the new line

Reply:
Thanks for noticing! Created a PR to address this: https://github.com/gatsbyjs/gatsby/pull/23140